docs/user/validation.md

```
@@ -0,0 +1,336 @@
+
+```bash
+aicr validate ... --fail-on-error=false
+```
+
+## Troubleshooting
+
+### Readiness pre-flight fails
+
+The CLI logs each readiness constraint comparison before any phase runs:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)